platform/darwin/src/MGLRuntimeStylingTests.m.ejs

```diff
@@ -0,0 +1,39 @@
+<%
+  const type = locals.type;
+  const layoutProperties = locals.layoutProperties;
+  const paintProperties = locals.paintProperties;
+-%>
+// This file is generated. 
+// Edit platform/darwin/scripts/generate-style-code.js, then run `make style-code-darwin`.
+
+#import "MGLStyleLayerTests.h"
+
+@interface MGL<%- camelize(type) %>LayerTests : MGLStyleLayerTests
+@end
+
+@implementation MGL<%- camelize(type) %>LayerTests
+
+- (void)test<%- camelize(type) %>Layer {
+    NSURL *geoJSONURL = [NSURL URLWithString:@"https://dl.dropboxusercontent.com/u/5285447/amsterdam.geojson"];
```

Comment:
On second thought, that would require mbgl to recognize the local file URL. Maybe we should [support asset: URLs](https://github.com/mapbox/mapbox-gl-native/blob/c354ae523a00b4c0e0577c4784b53553ea1a1cb7/platform/ios/src/MGLMapView.mm#L355-L359) here?